sdk/servicebus/azure-servicebus/setup.py

```diff
@@ -69,5 +69,6 @@
         'azure-core<2.0.0,>=1.14.0',
         "isodate>=0.6.0",
         "six>=1.11.0",
+        "typing-extensions>=3.7.4.3",
```

Comment:
qq @annatisch: is this the right place to put this? or should this be elsewhere?

Reply:
Should only be need for <3.8:
https://github.com/Azure/azure-sdk-for-python/blob/feature/communication-CallingServer-preview/sdk/communication/azure-communication-chat/setup.py#L59-L67